client-course-schedulizer/src/utilities/interfaces/dataInterfaces.ts

```diff
@@ -48,29 +70,29 @@ export interface Course {
   name: string;
   number: string;
   prefixes: string[];
+  sections: Section[];
   studentHours: number;
 }
 
 export interface Section {
   anticipatedSize: number;
   comments: string;
-  course: Course;
   // Overrides Course value
   facultyHours?: number;
   globalMax: number;
-  half: Half;
   instructors: Instructor[];
   letter: string;
   localMax: number;
   // Multiple Meetings possible if time/room differs on different days
   // Asynchronous classes should have an empty array of meeting times
   meetings: Meeting[];
+  semesterLength: SemesterLength;
   // Overrides Course value
   studentHours?: number;
   term: Term;
-  year: number;
+  year: number | string;
 }
 
 export interface Schedule {
-  sections: Section[];
+  courses: Course[];
```

Comment:
So a Schedule is a series of Courses where each course can have multiple sections, right?

Reply:
Yep 👍